README.md

```diff
@@ -71,7 +71,15 @@ See the JSDoc markdown documentation for the returned `balena` object in [DOCUME
 Support
 -------
 
-If you're having any problem, please [raise an issue](https://github.com/balena-io/balena-sdk/issues/new) on GitHub and the balena team will be happy to help.
+If you face any issues, please [raise an issue](https://github.com/balena-io/balena-sdk/issues/new) on GitHub and the balena team will be happy to help.
+
+Deprecation policy
+------------------
+
+The latest (minor,patch) release of each major balena-sdk version will be supported for a year after the next major version gets released.
+After that point older SDK versions will be considered as deprecated and you are highly encouraged to upgrade to the new major.
+
+For example, we will continue to support balena-sdk v12.33.0, which is the latest version of the v12 release, for a year after balena-sdk v13.0.0 got released.
```

Comment:
current latest is 13 right?

Reply:
Indeed

Reply:
Would it be a bit clearer to phrase it in terms of major releases, instead of (minor,patch)? For example:
```
`balena-sdk` uses [semver versioning](https://semver.org/), with the concepts of
major, minor and patch version releases.

The previous major version of balena-sdk will be supported for one year from the
date when the next major version is released. For example, balena-sdk v12.x.x
will be supported for one year from the date when v13.0.0 is released.

At the end of this support period, the older major version is considered deprecated
and you are highly encouraged to upgrade to the to latest major version.
```
Is there a meaningful difference between the following two statements? -
* Version `12.33.4` will be supported for a year from the date when v13 is released.
* Version `12.x.x` will be supported for a year from the date when v13 is released.

I have the impression that users would understand any of these two statements.

Reply:
I agree with Paulo here 👍

Reply:
Loved your suggestion @pdcastro :+1: 

> Is there a meaningful difference between the following two statements? -

There is some difference indeed, that being that we will only support the latest release of that previous major. For example v12 was initially using the image maker endpoints which we want to deprecate, but changing to the respective API endpoints was only a minor release.

So will do some slight modification on your proposal to also include the above concept (as discussed in the arch call). Thanks for this, it was really a big improvement.